utils.js

```diff
@@ -1,7 +1,7 @@
 // baseline estimates, used to improve performance
 var TX_EMPTY_SIZE = 4 + 1 + 1 + 4
 var TX_INPUT_BASE = 32 + 4 + 1 + 4
-var TX_INPUT_PUBKEYHASH = 106
+var TX_INPUT_PUBKEYHASH = 107
```

Comment:
This is variable.
I think the "average" range is going to be about 106-108,  but,  variable.